rpc/flipt/analytics/analytics.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package flipt.analytics;
+
+option go_package = "go.flipt.io/flipt/rpc/flipt/analytics";
+
+message GetFlagEvaluationsCountRequest {
```

Comment:
It may make sense to change this to `GetFlagEvaluationsStatsRequests/Response` to enable multiple statistics in the same RPC. I can think of `avg`, `p99`, etc.

Wdyt? Would it make sense to have a general stats endpoint that we can add new values to over time instead of having to add more methods for each stat? Or are counts different enough from histogram type metrics that we may need at least 2 different RPC calls?

Reply:
I would keep counts and duration quantiles as separate RPCs personally, as they're discretely separate measurements that will be served by separate queries.
This will be useful from an understanding the performance of these queries themselves in isolation from one another.
Otherwise, we could end up with one beast of an RPC call that does all the things.